app/classifier/tasks/drawing/summary.cjsx

```diff
@@ -19,14 +19,16 @@ module.exports = React.createClass
     expanded: @props.expanded
 
   render: ->
+    marks = (mark for mark in @props.annotation.value)
```

Comment:
sorry I'm prob missing something here, but what does `marks` give us that `@props.annotation.value` doesn't?